sims/vcs/dramsim2_ini

```diff
@@ -0,0 +1 @@
+../../generators/testchipip/src/main/resources/dramsim2_ini
```

Comment:
Is there anyway for this file not to exist in the sim. directories?

Reply:
Maybe refer to it from within the `generated-src` and have it copied in there?

Reply:
This was discussed in earlier comments.  The issue is that `ini_dir` is currently hardcoded to a relative path in the dramsim2 wrapper, so although the INI file is already copied (uselessly at the moment) as a resource, setting `ini_dir` to the `generated-src` directory requires a bit of contortion.  Symlinking is the simplest approach for now.

Reply:
Sorry I jumped straight into the code and didn't see the main comments.